build.fsx

```diff
@@ -75,6 +75,9 @@ let buildMergedDir = buildDir @@ "merged"
 let paketFile = buildMergedDir @@ "paket.exe"
 
 Environment.CurrentDirectory <- __SOURCE_DIRECTORY__
+
+System.Net.ServicePointManager.SecurityProtocol <- unbox 192 ||| unbox 768 ||| unbox 3072 ||| unbox 48
```

Comment:
You should avoid magic number and assign them to constants before ORing them.

Reply:
@haf Yes good eyes ;) 
I quickly copy&pasted it from `Utils.fs`. Basically, I think we need to fix this in either Octokit or FAKE